sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -39,6 +40,7 @@ export class AppConfigurationClient {
 
 // @public
 export interface AppConfigurationClientOptions {
+    httpClient?: HttpClient;
```

Comment:
Might be a blessing in disguise that we haven't added this to app config yet. It might be possible to migrate to corev2 without any breaking changes, not even little ones.

Reply:
Hmmm, with that logic we'll only migrate core-v2 packages to use the new recorder.
And wait for the core-v1 package to be migrated to core-v2 before migrating to the new recorder.

?